scripts/create_compdb.py

```diff
@@ -57,10 +61,23 @@ def _build_generated_files(bazel: str) -> None:
 
 
 def main() -> None:
+    parser = argparse.ArgumentParser(
+        description="Generates compilation data used by clangd.",
+        allow_abbrev=False,
+    )
+    parser.add_argument(
+        "--alsologtostderr",
+        action="store_true",
+        # type=bool,
+        default=False,
```

Comment:
I would skip the commented out bit, and the default is already provided by this action from my understanding.

```suggestion
```